@kiva/kv-components/vue/KvSideSheet.vue

```diff
@@ -184,8 +200,16 @@ export default {
 							transition: 'all 0.5s ease-in-out',
 						};
 					}, 10);
-				} else {
-					modalStyles.value = {};
+				} else if (stickyControls.value) {
```

Comment:
When does this `else` get hit? I'm confused why we need this in the `else`.

Reply:
So this gets called again when the side sheet is opened? We'll want to make sure the animation respects the navigation height/location.